debian/changelog

```diff
@@ -0,0 +1,5 @@
+pymavlink (2.4.41) UNRELEASED; urgency=medium
```

Comment:
Is this the changelog for the package or the packaged software?

Reply:
hi,
this is a changelog for the package itself... In most cases should be aligned with the actual software version